src/MSBuild/LiveLogger/LiveLogger.cs

```diff
@@ -360,10 +366,20 @@ private void ProjectFinished(object sender, ProjectFinishedEventArgs e)
                         {
                             // Ignore any GetDirectoryName exceptions.
                         }
+
+                        // If the output path is under the initial working directory, make the console output relative to that to save space.
+                        if (outputPathSpan.StartsWith(_initialWorkingDirectory.AsSpan(), FileUtilities.PathComparison)
+                            && (outputPathSpan[_initialWorkingDirectory.Length] == Path.DirectorySeparatorChar
```

Comment:
It's not possible for `outputPathSpan` to be the same as `_initialWorkingDirectory` to cause a crash here, is it?

Reply:
Hm. Not normally, but we can't control what projects might emit, so we should definitely guard against this.